### ISSUE TITLE
Houdini integration broken

### ISSUE THREAD

**Issue by mruegenberg** [cgwire/dccutils#7]
**Context**
Hello,
I've been looking at the Houdini integration and found that in houdini.py currently a bunch of methods (e.g the software name) are missing while others look like they might be superfluous/old. Others (eg for grabbing renders) have different arguments for eg the colour space stuff.

Happy to help out with testing and some development.

**Comment by frankrousseau** [comment]
Yes, we didn't work on the Houdini integration for a while. It needs some cleanup.
